src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationClientSideProviders/MS/Internal/AutomationProxies/WindowsEditBoxRange.cs

```diff
@@ -203,10 +203,7 @@ ITextRangeProvider ITextRangeProvider.FindText(string text, bool backwards, bool
             // A null string is not should throw an ArgumentNullException while an empty string should throw an ArgumentException.
             // Therefore we can not use IsNullOrEmpty() here, suppress the warning.
 #pragma warning suppress 6507
-            if (text == null)
-            {
-                throw new ArgumentNullException("text");
-            }
+            ArgumentNullException.ThrowIfNull(text);
```

Comment:
This could even be `ArgumentException.ThrowIfNullOrEmpty(text)` if the maintainers are willing to slightly change the error message for the empty case

Reply:
@halgab yes, we can change it to ```ArgumentException.ThrowIfNullOrEmpty(text)``` 
We are considering this PR for this month testing. If you agree, I would be happy to add this change.

Reply:
I can add the commit right now if you want, but if it's easier for you to handle it I can let you do it.

Reply:
done

Reply:
I have noticed another place exactly like this one and updated it as well